spec/factories/grda_warehouse/client_notes/grda_warehouse_client_notes_alerts.rb

```diff
@@ -0,0 +1,29 @@
+FactoryBot.define do
+  factory :grda_warehouse_client_notes_expired_alert, class: 'GrdaWarehouse::ClientNotes::Alert' do
+    association :client, factory: :grda_warehouse_hud_client
+    user
+    note { 'Test' }
+    expiration_date { Date.yesterday }
+  end
+
+  factory :grda_warehouse_client_notes_active_alert, class: 'GrdaWarehouse::ClientNotes::Alert' do
+    association :client, factory: :grda_warehouse_hud_client
+    user
+    note { 'Test' }
+    expiration_date { Date.tomorrow }
```

Comment:
I haven't been able to verify if Date.yesterday and Date.tomorrow are timezone aware.  If they are not, these might cause test failures when GMT is a different date from when we run the tests.  `Date.current` **is** timezone aware, so you can always do something like `Date.current - 1.days`